### ISSUE TITLE
Meet Section 508 1922 (k) - when unable to comply, provide a text-only page

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#558]
## Summary
A text-only page, with equivalent information or functionality, shall be provided to make a web site comply with the provisions of this part, when compliance cannot be accomplished in any other way. The content of the text-only page shall be updated whenever the primary page changes.
**Conformance Level:** N/A
**Existing 508 Corresponding Provision:** 1922 (k)

**Comment by joshdanielson** [comment]
No features require an alternative page for accessibility. Closing.
